atlascope/core/migrations/0007_auto_20220606_1603.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 3.2.13 on 2022-06-06 16:03
```

Comment:
I think best practice is to rename these "auto" migration files to reflect what change they are associated with. (Do I have that right, @annehaley?)